base/applications/drwtsn32/main.cpp

```diff
@@ -89,6 +97,16 @@ static void PrintThread(FILE* output, DumpData& data, DWORD tid, ThreadData& thr
 #if defined(_M_IX86) || defined(_M_AMD64)
         xfprintf(output, "dr0:%p dr1:%p dr2:%p dr3:%p dr6:%p dr7:%p" NEWLINE,
                  ctx.Dr0, ctx.Dr1, ctx.Dr2, ctx.Dr3, ctx.Dr6, ctx.Dr7);
+#elif defined(_M_ARM)
+        for (int n = 0; n < ARM_MAX_BREAKPOINTS; ++n)
+            xfprintf(output, "Bvr%d:%p%s", n, ctx.Bvr[n], ((n + 1) == ARM_MAX_BREAKPOINTS) ? NEWLINE : " ");
+        for (int n = 0; n < ARM_MAX_BREAKPOINTS; ++n)
+            xfprintf(output, "Bcr%d:%p%s", n, ctx.Bcr[n], ((n + 1) == ARM_MAX_BREAKPOINTS) ? NEWLINE : " ");
```

Comment:
Windbg only shows `Bcr[0]` and `Bcr[1]`

Reply:
![image](https://user-images.githubusercontent.com/610685/120905391-1660e980-c652-11eb-9c7e-d2210ff6c156.png)